### PR TITLE
Simpler getting started

### PR DESCRIPTION
**Pull request by skim1420**
@duftler @rbuskens ptal

### CODE REVIEW

File: documentation/getting_started.md [hunk 1]
```diff
@@ -177,43 +168,34 @@ Create an AWS virtual machine.
...
-        gcloud config set project powerful-surf-12345
-        gcloud config set compute/zone us-central1-f
+1. Click the Launch on Google Cloud Platform button
 
-1. Create a Google Compute Engine virtual machine. This command assumes you
-   set the <code>gcloud</code> defaults above.
+1. Select the project you selected in the GCP Setup step above and click the Continue button
 
-        gcloud compute instances create spinnaker-test --image ubuntu-14-04 --machine-type n1-standard-8 --scopes compute-rw
+1. Keep defaults and click the Deploy Spinnaker near the bottom
```
Comment (duftler) [inline documentation/getting_started.md]: Maybe drop the 'the' or add 'button' after 'Deploy Spinnaker'.

File: documentation/getting_started.md [hunk 2]
```diff
@@ -223,8 +205,6 @@ components, including Redis and Cassandra, which Spinnaker components
 use to store data. If you see any errors, please just run the command
 again.
 
-Note that it can take several minutes for Spinnaker to start.
-
 It will take several minutes to install and configure Spinnaker along with
```
Comment (tomaslin) [inline documentation/getting_started.md]: This feels like it belongs before running Spinnaker on a host not on AWS or GCP

**Comment by duftler** [comment]
I think this is a big improvement. LGTM.
@dstengle @dpeterka fyi

File: documentation/getting_started.md [hunk 3]
```diff
@@ -177,43 +168,34 @@ Create an AWS virtual machine.
...
+    for post-configurations to complete.
 
-## Step 3: Install and run Spinnaker
-(Note: If you are running on AWS and provisioned an instance using one of the pre-baked Spinnaker-Ubuntu AMIs,
-you can skip ahead to Step 4.)
+### Running Spinnaker on a host not on AWS or GCP
 
-If you have chosen to run Spinnaker inside an Amazon Web Services or
-Google Compute Engine VM, please ssh into the VM.
+If you have chosen to run your Spinnaker instance somwhere other than on AWS or GCP (e.g. your local workstation),
```
Comment (tomaslin) [inline documentation/getting_started.md]: This section seems to appear out of nowhere and there is no reference to it. It feels like it should say "If you just want to try running on your local workstation, use this command". Also, would be nice if there was a link to the spinnaker/spinnaker scripts here.

**Comment by duftler** [comment]
Sorry @tomaslin, missed your comment. I'll revert so it can be addressed.

File: documentation/getting_started.md (hunk 2 above)
Comment (tomaslin) [inline documentation/getting_started.md]: This line feels like it should be before the Running on non aws section. I also now don't know which port my localhost/workstation instance should be ran on

File: documentation/getting_started.md [hunk 4]
```diff
@@ -10,26 +10,21 @@ lang: en
 
 # Getting started
 
-These instructions quickly get Spinnaker up and running on an Amazon
-Web Services (AWS) EC2 or Google Compute Engine virtual machine
-(VM) or local machine, then walk you through the steps of setting up
-and using Spinnaker to deploy to and manage clusters on AWS and/or
-Google Cloud Platform (GCP).
+These instructions quickly get Spinnaker up and running, then walk you through
```
Comment (tomaslin) [inline documentation/getting_started.md]: should be a link back to spinnaker/spinnaker here.

Comment (skim1420) [inline documentation/getting_started.md]: Link from which part of text? Or what context?

**Pull request merged**